index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
           </div>
         </section>
     </main>
+    <div class="main-body">  
+      <section class="main-row">
+        <h1>My Recent work</h1>
+        <div class="h-line"><hr></div>
+        <div class="story-section">
+          <img src="images/st-img.png" alt="st-imag">
+          <div class="h1-tag"><h1>Multi-Post Stories</h1></div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div class="lang-btn">
+          <button type="button">css</button>
+          <button type="button">html</button>
+          <button type="button">bootstrap</button>
+          <button type="button">Ruby</button>
+        </div>
+        <div class="see-prjct-btn">
+          <button type="button">See Project</button>
```

Comment:
- [ ] Let's try to remove the space under the button, this way it matches the Figma template.

Yours:
![image](https://user-images.githubusercontent.com/79114233/139542348-0278a413-bfd2-4c3b-a12e-f32b98956728.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/139542357-ed31d679-e457-480a-9701-86da3b06ad78.png)